.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - "3.6.9"
+before_script:
```

Comment:
Non conoscevo travis ed è la prima volta che lo leggo/uso, quindi perdona domande stupide: l'environment potremmo crearlo anche in un `install` phase?

Reply:
credo che la fase di `install` sia più per installazione di pacchetti di sistema, però credo non ci siano grandi differenze, dovrebbe essere solo una questione logica. Se preferisci provo

Reply:
A livello logico mi sembra più pulito, se non ci guadagniamo niente in termine di pulizia e ordine della CI lasciamo pure così

Reply:
ok mi informo

Reply:
Funziona tutto, mergio